infrastructure/main.tf

```diff
@@ -0,0 +1,87 @@
+provider "azurerm" {}
+
+provider "vault" {
+  address = "https://vault.reform.hmcts.net:6200"
+}
+
+locals {
+  vault_section = "${var.env == "prod" ? "prod" : "test"}"
+}
+
+data "vault_generic_secret" "s2s_secret" {
+  path = "secret/${local.vault_section}/ccidam/service-auth-provider/api/microservice-keys/platformJobScheduler"
```

Comment:
@rkondratowicz raised a question whether it's a correct usage and should it not be e.g. `platform-job-scheduler` instead. @jasonpaige ?

Reply:
iirc that was the same naming convention of what was already there in vault for the other apps... but yeh kind of annoying I must admit. We should correct it when we migrate to Azure Key Vault I think!